sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -241,24 +241,24 @@ namespace Azure { namespace Storage { namespace Blobs {
      * @param bufferSize Size of the memory buffer. Size must be larger or equal to size of the blob
      * or blob range.
      * @param options Optional parameters to execute this function.
-     * @return A DownloadBlobToResult describing the downloaded blob.
+     * @return A DownloadToBlobResult describing the downloaded blob.
      */
-    Azure::Core::Response<DownloadBlobToResult> DownloadTo(
+    Azure::Core::Response<DownloadToBlobResult> DownloadTo(
         uint8_t* buffer,
         std::size_t bufferSize,
-        const DownloadBlobToOptions& options = DownloadBlobToOptions()) const;
+        const DownloadToBlobOptions& options = DownloadToBlobOptions()) const;
```

Comment:
DownloadToBlob is really misleading. I would rather have inconsistency than having this name.

Reply:
Do you think `UploadFromBlockBlob` is misleading?

Reply:
Yes, very. `UploadBlockBlobFrom` is way better.

Reply:
This is proposed by Jeffrey in the Email thread. Why not bring it up if you're not okay with it?

Reply:
Email sent to get Jeffrey's opinion on this.

Reply:
fixed